grails-app/conf/Config.groovy

```diff
@@ -263,7 +263,8 @@ portal {
 
         supplementary = [
             "WWW:LINK-1.0-http--link",
-            "WWW:LINK-1.0-http--downloaddata"
+            "WWW:LINK-1.0-http--downloaddata",
+            "WWW:LINK-1.0-http—related"
```

Comment:
Yeah, this seems to be incorrect. Perhaps a text editor somewhere along the way turned a double hyphen into a dash of some kind.